server/api/fileOperations.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import Router from "koa-router";
+import { NotFoundError } from "../errors";
+import auth from "../middlewares/authentication";
+import { FileOperation, User, Collection, Team } from "../models";
+import policy from "../policies";
+import { presentFileOperation } from "../presenters";
+import { getSignedUrl } from "../utils/s3";
+import pagination from "./middlewares/pagination";
+
+const { authorize } = policy;
+const router = new Router();
+
+router.post("fileOperations.list", auth(), pagination(), async (ctx) => {
```

Comment:
If we add a `fileOperations.info` it will allow polling the state of the export via the API, automating data export is actually a surprisingly common usecase so I do believe it would be used.